pkgs/tools/system/hostctl/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildGoModule, fetchFromGitHub, lib, installShellFiles }:
+
+buildGoModule rec {
+  pname = "hostctl";
+  version = "1.0.14";
+
+  src = fetchFromGitHub {
+    owner = "guumaster";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "02bjii97l4fy43v2rb93m9b0ad8y6mjvbvp4sz6a5n0w9dm1z1q9";
+  };
+
+  vendorSha256 = "1lqk3cda0frqp2vwkqa4b3xkdw814wgkbr7g9r2mwxn85fpdcq5c";
+
+  buildFlagsArray = [ "-ldflags=-s -w -X github.com/guumaster/hostctl/cmd/hostctl/actions.version=${version}" ];
+
+  nativeBuildInputs = [ installShellFiles ];
+  postInstall = ''
+    installShellCompletion --cmd hostctl \
```

Comment:
This style still isn't able to be used yet as it's in `staging-next.

It was briefly on master due to a bad merge that included commits from `staging`/`staging-next`.

Reply:
Oh thank you! @zowoq &mdash; how much time can we expect to pass before this becomes available on master? &mdash; so I can put a mark in my calendar and rebase...

Reply:
There is no fixed time window, but you can follow FRidh's PRs. The current one is https://github.com/NixOS/nixpkgs/pull/101207.

Reply:
I think a better approach is to rebase this on `staging-next`, do your review/testing/etc now while you're in context (if possible), and merge to `staging-next`.  At least unless some `staging-next` breakage is blocking this or such complications...